README.md

```diff
@@ -10,8 +10,7 @@ A simple subclass of harmony's Text that counts up or down. Customize the counti
 
 See the demo project for more implementation details.
 
-<img src="https://github.com/priyankabb153/TextCounter/blob/main/screenshots/TextCounterDemo.gif" width="256">
-
+<img src="screenshots/TextCounterDemo.gif" width="256">
```

Comment:
Can increase the text size in the gif. Currently it is too small for demo in tablet

Reply:
Done